### PR TITLE
Update yeoman

### PR DESCRIPTION
**Pull request by zckrs**

### CODE REVIEW

File: app/src/write.js [hunk 1]
```diff
@@ -31,13 +18,18 @@ module.exports = function(GulpAngularGenerator) {
    * Pass through each files and actually copy them
    */
   GulpAngularGenerator.prototype.writeFiles = function writeFiles() {
-    var process = processor(this);
+    this.props.angularModulesObject = {};
+    this.props.angularModules.map(function (module) {
+      this[module.key] = module.module;
+    }, this.props.angularModulesObject);
```
Comment (zckrs) [inline app/src/write.js]: @Swiip here I create a new `props` used in  `_bower.json` to replace `_.findWhere()` in template.

**Comment by Swiip** [comment]
According to build logs, even if it works, don't run installs (npm and bower) on inception seams to be broken.

**Comment by zckrs** [comment]
Ready to merge IMO

**Comment by Swiip** [comment]
:+1:

**Pull request merged**